dodam-student/src/main/kotlin/com/b1nd/dodam/student/MainActivity.kt

```diff
@@ -73,43 +73,44 @@ class MainActivity : ComponentActivity() {
                     isLogin = datastoreRepository.token.first().isNotEmpty()
                 }
             }
-
-            DodamTheme {
-                isLogin?.let {
-                    DodamApp(
-                        isLogin = it,
-                        logout = {
-                            lifecycleScope.launch {
-                                datastoreRepository.deleteUser()
-                                finish()
-                            }
-                        },
-                        firebaseAnalytics = firebaseAnalytics,
-                        firebaseCrashlytics = firebaseCrashlytics,
-                    )
-                } ?: run {
-                    Box(
-                        modifier = Modifier
-                            .background(MaterialTheme.colorScheme.background)
-                            .fillMaxSize(),
-                    ) {
-                        Icon(
-                            modifier = Modifier
-                                .size(180.dp)
-                                .align(Alignment.Center),
-                            imageVector = DodamLogo,
-                            contentDescription = null,
-                            tint = MaterialTheme.colorScheme.primary,
+            com.b1nd.dodam.designsystem.DodamTheme {
+                DodamTheme {
```

Comment:
why DodamTheme in com.b1nd.dodam.designsystem.DodamTheme?

Reply:
Currently, Dodam Student is using DDS Version 0.1.2, and the OnBoarding screen is using DDS Version 1.0.0[CMM] for both student and teacher apps.
There is still not enough time to replace the design system version of Dodam Student, so we decided to modify the screen only for modules that both Teacher and Student use together